clean-code/README.md

```diff
@@ -0,0 +1,26 @@
+## Clean code example for Fiber and PostgreSQL
```

Comment:
pls add a link in the README.md in root folder to this file

Reply:
Added